src/App.js

```diff
@@ -23,7 +44,19 @@ export default function App() {
     setActivePage('cropdetails')
   }
 
-  function handleClickBack() {
+  function handleClickList() {
```

Comment:
maybe no need to put this in an extra function but if it makes your code more readable it is good to keep it